src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx

```diff
@@ -126,14 +134,37 @@ const ValidatorsList = ({ onCheckboxClickHandler, selectedValidators, selectUnse
   //   });
   // };
 
+  const copyToClipboard = (publicKey: string) => {
+    navigator.clipboard.writeText(publicKey);
+    notificationsStore.showMessage('Copied to clipboard.', 'success');
+  };
+
+  const openBeaconcha = (publicKey: string) => {
```

Comment:
openBeaconcha and openExplorer should be the same function .something like openLink(url, googleTagData)

Reply:
done